deployment/stacks/api.ts

```diff
@@ -61,8 +61,7 @@ export function createApiComponents(props: CreateApiComponentsProps) {
   );
 
   const logGroup = new logs.LogGroup(scope, "ApiAccessLogs", {
-    logGroupName: `/aws/api-gateway/${stage}-app-api`,
```

Comment:
Why are we no longer naming this log group?

Reply:
cdk will automatically name it in a way that makes it easier to understand that it contains api access logs. It will also add characters to the end of the name so that you can deploy it 8 times and even if the log group is retained the names won't clash.